confdgnmi/src/confd_gnmi_common.py

```diff
@@ -8,6 +9,13 @@
 PORT = 50061
 
 
+class PayloadEncoding(Enum):
```

Comment:
I think it is not worth to introduce this enum, if you assign values from  `gnmi_pb2.` (or other gnmi library), In this way adapter dependency on `gnmi_pb2.` encoding is not avoided, so  just return `gnmi_pb2.`  values as list.

Reply:
I don't know. If we want to shield lower layers from gNMI stuff, this works about as well as anything else. But I still cannot make up my mind whether any shielding is necessary.

Reply:
yes, I agree, but in case of shielding, `gnmi_pb2` should not be used in assignment here. It should be moved up, at least to server code.

Reply:
removed the "shielding" enum